Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Xamarin.Forms.Controls.Issues.Shared.projitems

```diff
@@ -1500,7 +1501,7 @@
   <ItemGroup>
     <EmbeddedResource Include="$(MSBuildThisFileDirectory)Issue7593.xaml">
       <SubType>Designer</SubType>
-      <Generator>MSBuild:Compile</Generator>
+      <Generator>MSBuild:UpdateDesignTimeXaml</Generator>
```

Comment:
Is this intentional?

Reply:
Nope! :( it happens automatically whenever I add something I think. Probably some difference between how VSWin and VSMac handles things. I don't think it has any impact but I can take it out if you want